content/docs/start/data-pipelines.md

```diff
@@ -8,8 +8,8 @@ version, and reproduce your data science and machine learning workflows.'
 
 Versioning large data files and directories for data science is great, but not
 enough. How is data filtered, transformed, or used to train ML models? DVC
-introduces a mechanism to capture _data pipelines_ — series of data processes
-that produce a final result.
+introduces a mechanism to capture <abbr>data pipelines</abbr> — a series of data
+processes which produce a final result.
```

Comment:
Sorry, it just reads strange to me ("that produce a final result" was prob. meant as a restrictive clause) but maybe I have a bias. Let's ask someone else to check some of the "that" -> "which" changes in this PR...

Cc @dberenbaum @jendefig could one of you maybe take a quick look and opine on "that" vs "which" specifically? Thanks

Reply:
Thanks @casperdcl and @jorgeorpinel!

In this particular case, I agree that it seems like a restrictive clause, in which case "that" is appropriate. I'm also confused by the change from "series" to "a series" since it seems that "data pipelines" refers to more than one series (maybe the end of the sentence should be changed from "a final result" to "final results" so that it's also plural 🤔 ).

I haven't looked through the whole PR, but IMO we should lean towards leaving docs as is unless there's a clear justification provided for why to make the change. It's unrealistic to expect an explanation for every typo, so hopefully we can use judgment and include explanation whenever we think it's not immediately obvious why the change is being suggested.

Reply:
Thanks @dberenbaum 

> haven't looked through the whole PR

Yeah no need 🙂 we'll figure it out. Feel free to unsubscribe, sorry for the millions of notifications you prob. got after my initial mention.

Reply:
The paragraph at the start of the "Pipeline stages" section is worded much better (less ambiguous). Perhaps:

```suggestion
introduces a mechanism to capture <abbr>data pipelines</abbr> — series of data
processes, each producing one or more results.
```

Reply:
> each producing one or more results

I like that there's no "that" or "which" 😆 but it's not clear why "each" as it's not a finite set but more the concept of pipelines we're talking about. I rolled this one back for now.